src/api/Areas/Services/Controllers/ServerItemController.cs

```diff
@@ -106,16 +106,17 @@ public IActionResult Add(ServerItemModel model)
     ///
     /// </summary>
     /// <param name="model"></param>
+    /// <param name="updateTotals"></param>
     /// <returns></returns>
     [HttpPut("{id}", Name = "UpdateServerItem-Services")]
     [Produces(MediaTypeNames.Application.Json)]
     [ProducesResponseType(typeof(ServerItemModel), (int)HttpStatusCode.OK)]
     [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
     [SwaggerOperation(Tags = new[] { "Server Item" })]
-    public IActionResult Update(ServerItemModel model)
+    public IActionResult Update(ServerItemModel model, bool updateTotals = false)
     {
         var entity = model.ToEntity();
-        _serverItemService.Update(entity);
+        _serverItemService.Update(entity, updateTotals);
```

Comment:
Moved the logic of calculating the total storage from the file system items to the server.